docs/basics/integrations/deque.md

```diff
@@ -0,0 +1,137 @@
+---
+id: deque
+title: "Deque axe™ Integration"
+sidebar_label: "Axe Deque"
+keywords:
+- accessibility-testing
+- accessibility
+- automated-testing
+- how-to
+---
+
+<p><small><span class="highlight beta">BETA</span></small></p>
+
+[Deque's axe™](https://www.deque.com/axe/) is one of the world's leading digital accessibility toolkits. The Sauce Labs integration allows you to run your accessibility tests, using axe™, on our platform.
+
+## What You'll Need
+* [A Sauce Labs Account](https://saucelabs.com/sign-up)
+* [A Deque Account (Optional)](https://axe.deque.com/plans)
+* [Install Git](https://git-scm.com/downloads)
+* [Install Node/NPM](https://nodejs.org/en/download/)
+
+## Project Setup
+
+Clone or download the [example project](https://github.com/saucelabs/axe-wdio):
+
+```bash
```

Comment:
It probably doesn't matter, but are we assuming that anyone who is using SSL will know what they are doing and won't care that this code sample is HTTPS?

Reply:
yea it doesn't matter, that's why I put the link to the project and the sample code (copied from github). They should be able to figure it out if they're going to run the test.